src/modules/tableValues.js

```diff
@@ -29,11 +40,31 @@ const getFormattedCellValue = ({ value, header = {}, visualization = {} }) => {
     }
 
     if ([VALUE_TYPE_DATE, VALUE_TYPE_DATETIME].includes(header.valueType)) {
+        let valueType = header.valueType
```

Comment:
Maybe move this above the `if` and use it in the `includes` as well

Reply:
On line 72 I see `header?.valueType`, so maybe figure out if the `?` is needed everywhere or could be removed